### DISCUSSION TITLE
Integration of the blog with medium.com [does this make any sense?]

### DISCUSSION THREAD

**Discussion by balansczerni** [squidfunk/mkdocs-material#5981] [Ideas]
<sub>Disclaimer: I don't know much about SEO or programming. I just came up with an idea that may not make much sense, but I'm curious about your input if you have a moment.</sub>

**I'm wondering about blog integration with something like medium.com.**

I imagine it this way: 

* I attach a script to my MkDocs (example.com) with a blog build-in plugin, (which, in addition to the texts contained in the /posts/ directory), also downloads texts from the blog located on medium.com (e.g. under subdomain like blog.example.com or regular medium.com).
* It downloads the thumbnail, title, description and possibly metadata for display in the blog post list, but replaces the URL/redirects already for the entry on blog.example.com.
* Maybe it also marks such a post with label "read on medium" or sth.

I wonder if this has any benefits, e.g. for SEO or is it counter-effective.

**Comment by squidfunk** [comment]
Thanks for asking!

I'm not sure Medium allows that, as they're locking down more and more. Medium is very well known for notoriously crippling the user experience to force users into their premium membership. In fact, the blog plugin was started as a contender for Medium, as it allows you to create a blog that you can host for free on GitHub Pages, that has great SEO and for which you will always own the data, given that Material for MkDocs is and always will be Open Source.

That being said, it would be awesome to find out an easy way to migrate your content from Medium to the blog plugin. This is what many users are doing right now, given that the blog plugin can already be considered stable and mature, and is actively maintained. However, what I'm proposing is probably the other way around of what you're imagining.
